samples/graph-notification-team-channel/csharp/GraphNotification/Manifest/manifest.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.12",
+  "version": "1.0.0",
+  "id": "<< Manifest ID >>",
+  "packageName": "com.teams.channel.notification",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.teams.com/privacy",
+    "termsOfUseUrl": "https://www.teams.com/termsofuser"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Notifications",
+    "full": "Notifications"
+  },
+  "description": {
+    "short": "This sample uses subscription that will post notifications.",
```

Comment:
Get change notifications for teams and channels using Microsoft Graph.

Reply:
fixed